src/app/signOut/signOut.js

```diff
@@ -9,7 +9,13 @@ const logger = require('./../../infrastructure/logger');
 
 const signUserOut = (req, res) => {
   if (req.user.id_token) {
-    logger.audit('User logged out', { userId: req.user.sub, email :req.user.email });
+    logger.audit('User logged out', {
+      type: 'Sign-out',
+      subType: 'services',
```

Comment:
subType can be omitted. It is optional and you already define services as the client